src/output-json-mdns.c

```diff
@@ -0,0 +1,212 @@
+/* Copyright (C) 2025 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+#include "suricata-common.h"
+#include "conf.h"
+
+#include "threadvars.h"
+
+#include "util-debug.h"
+#include "app-layer-parser.h"
+#include "output.h"
+
+#include "output-json.h"
+#include "output-json-mdns.h"
+#include "output-json-dns.h"
+#include "rust.h"
+
+/* Using SCDnsLogFileCtx and SCDnsLogThread from output-json-dns.h */
+
+bool AlertJsonMdns(void *txptr, SCJsonBuilder *js)
+{
+    SCDnsLogConfig config = {
+        .version = DNS_LOG_VERSION_DEFAULT,
+        .flags = DNS_LOG_FORMAT_DETAILED | DNS_LOG_REQUESTS | DNS_LOG_RESPONSES |
+                 DNS_LOG_ALL_RRTYPES,
+        .log_additionals = true,
+        .log_authorities = true,
+        .answers_in_request = true,
+    };
```

Comment:
Maybe when part of an alert, everything should be on.. Full verbose mode. This would mean the flags, rcode, TTL, opcode.